contracts/Liquidation.sol

```diff
@@ -152,11 +159,6 @@ contract Liquidation is ILiquidation, Ownable {
         address account
     ) internal returns (uint256) {
         require(amount > 0, "LIQ: Liquidation amount <= 0");
-        // Limits the gas use when liquidating
-        require(
-            tx.gasprice <= IOracle(tracer.gasPriceOracle()).latestAnswer(),
-            "LIQ: GasPrice > FGasPrice"
-        );
```

Comment:
So the assumption is we now check this logic in the Tracer itself?

Reply:
I've changed this to now check in Liquidation.sol **instead** of TracerPerpsswaps.sol. Since fastGasOracle is in Liquidation.sol